Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -850,12 +852,13 @@ void HGCalGeomParameters::loadWaferHexagon(HGCalParameters& php) {
       if (inr%2 == inc%2) {
         double xpos = nc*dx;
         double ypos = nr*dy;
-        xc[0] = xpos+dx; yc[0] = ypos-0.5*rr;
-        xc[1] = xpos+dx; yc[1] = ypos+0.5*rr;
-        xc[2] = xpos;    yc[2] = ypos+rr;
-        xc[3] = xpos-dx; yc[3] = ypos+0.5*rr;
+        xc[0] = xpos;    yc[0] = ypos+rr;
+        xc[1] = xpos-dx; yc[1] = ypos+0.5*rr;
+//       xc[2] = xpos-dx; yc[2] = ypos-0.5*rr;   Bug correction
```

Comment:
will this be fixed later?

Reply:
@jbsauvan FYI

Reply:
Hi @bsunanda could you explain a bit more what has been changed here that is affecting the trigger?

Reply:
@jbsauvan Hi the computation of 6 corners decides if the wafer is within the fiducial volume or not. In earlier version one of the corners was entered twice and one was missed out. This sometimes eliminate some wafers whose only one corner was within the fiducial volume. Now after correcting for this one gets some more cells for which L1Trigger geometry was not ready. This was done in 2 places and I can send the versions which have this bug fixed. But if I have the correct version in PR it will complain in L1Trigger and sends an exception. May be after the current PR is accepted I shall make a branch with corrected pre-TDR HGCal geometry and disable the exception at L1. You can take it up and make appropriate correction if needed.

Reply:
Ok, I see. I can produce an updated trigger geometry map and update the trigger config. Just send me a branch name or commit hash where you have the fix in and which I can use.
Thanks